crates/biome_js_formatter/src/js/auxiliary/template_element.rs

```diff
@@ -179,6 +216,10 @@ impl AnyTemplateElement {
             AnyTemplateElement::TsTemplateElement(template) => template.r_curly_token(),
         }
     }
+
+    fn has_new_line_in_range(&self) -> bool {
+        self.syntax().text().contains_char('\n')
```

Comment:
This won't work on Windows, where newlines can be `\r`. Also, wouldn't you risk hitting false positives, where you might have a string literal that contains a literal new line? e.g:

```js
const f = `some \n \n ${foo} \n bar`;
```

Wouldn't it be safer to extract the trivia and check for `TriviaKind::Whitespace` that contain newlines?

Reply:
Good point. Fixed in https://github.com/biomejs/biome/pull/2500/commits/43493faf97161624318576f077dd6a5d32e87747